packages/wow-ui/src/hooks/useCheckedState.ts

```diff
@@ -13,17 +16,44 @@ interface CheckedStateProps {
 const useCheckedState = ({
   defaultChecked = false,
   checked: checkedProp,
-  disabled,
+  disabled: disabledProp,
+  value,
   onChange,
   onClick,
   onKeyDown,
 }: CheckedStateProps) => {
-  const [checked, setChecked] = useState<boolean>(() =>
-    checkedProp !== undefined ? checkedProp : defaultChecked
-  );
+  const group = useContext(MultiGroupContext);
+  const groupOnChange = group.onChange;
```

Comment:
p4;
구조분해할당으로 group 관련 프로퍼티들을 가져오면 어떨까요?!
```suggestion
const { onChange: groupOnChange, checked: groupCheckedValues, disabled: groupDisabled } = useContext(MultiGroupContext);
```

Reply:
굿굿 좋습니다! 반영해둘게요~!